cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/schedule/AbstractSchedulerService.java

```diff
@@ -80,7 +83,8 @@ protected final void startScheduler() {
    */
   protected final void stopScheduler() {
     try {
-      delegate.stop();
+      timeScheduler.stop();
+      streamSizeScheduler.stop();
       LOG.info("Stopped scheduler");
     } catch (SchedulerException e) {
       LOG.error("Error stopping scheduler {}", e.getCause(), e);
```

Comment:
stopping what scheduler? You now have two schedulers, so the message should name which one.